frontend/public/metalkube/components/host/host.tsx

```diff
@@ -95,7 +98,9 @@ const HostRow = ({ obj: host }) => {
         />
       </div>
       <div className={statusColumnClasses}>
-        <BaremetalHostStatus host={host} />
+        <WithResources resourceMap={machineName ? hostResourceMap : {}}>
```

Comment:
I think it would be better to wrap the whole `HostList` with `WithResources` - it will result in only one websocket connection instead of one for each host, which is a lot better